shark_turbine/kernel/ops/wave_ops.py

```diff
@@ -38,6 +38,26 @@ def allocate(
     ...
 
 
+def cast(register: "Register", type: DataType) -> "Register":
+    ...
+
+
+def div(lhs: "Register", rhs: "Register") -> "Register":
+    ...
```

Comment:
The predeclarations here do not have CustomOps associated with them
We should map the `div` here to the Python operator like [here](https://github.com/iree-org/iree-turbine/blob/44c7634c3a0d4e26a8b31fc1ffa5bbb2cc8fbfcd/shark_turbine/kernel/ops/wave_ops.py#L310). The branch needs to be rebased for that